src/messages/MessageBuilder.cpp

```diff
@@ -385,8 +385,26 @@ EmotePtr makeAutoModBadge()
         Url{"https://dashboard.twitch.tv/settings/moderation/automod"}});
 }
 
-EmotePtr makeSharedChatBadge(const QString &sourceName)
+EmotePtr makeSharedChatBadge(const QString &sourceName, const QString &url)
 {
+    if (!url.isEmpty())
+    {
+        QString modifiedUrl = url;
+
```

Comment:
```suggestion
```

Reply:
Fixed in [`dbde6f3` (#5760)](https://github.com/Chatterino/chatterino2/pull/5760/commits/dbde6f3d3ef732778ff3738a7afcbeb57fee0ddb)